src/main/java/seedu/trippie/command/Command.java

```diff
@@ -1,4 +1,5 @@
 package seedu.trippie.command;
 
 public class Command {
+
 }
```

Comment:
This shouldve been an abstract class for all the other commands.

Reply:
I think creating this class as a parent for everything else will solve the parser error.